### ISSUE TITLE
Implement skip to content a11y links

### ISSUE THREAD

**Issue by ramboz** [hlxsites/delta#115]
The Delta webpage uses accessibility short links "Skip to content":

![Image](https://user-images.githubusercontent.com/1235810/228044807-cc902631-eadf-460f-8ea5-8be1dc67ea35.png)
![Image](https://user-images.githubusercontent.com/1235810/228044806-88da504a-d763-40dc-a08d-3e2540d27d09.png)

We need to implement those as well.
- links need to be out of viewport by default (not `display: none`) so that they are not visible to regular users but still picked up by assistive technologies
- when explicitly focused via "Tab", they need to show up in the viewport
- when clicked (or by pressing space on the keyboard) we need to scroll to the main content, and shift the focus to main content as well so the next "Tab" continues from the main heading (and does not continue in the header)
